dependabotAlerts.js

```diff
@@ -0,0 +1,120 @@
+"use strict";
+const _ = require("lodash");
+
+const Promise = require("bluebird");
+const token = process.env.GITHUB_TOKEN;
+const { Octokit } = require("@octokit/rest");
+const { sum } = require("lodash");
+const octokit = new Octokit({
+  auth: token,
+  // Set GitHub Auth Token in environment variable
+});
+
+async function getAlerts(repos) {
+  const sortedAlerts = [];
+  const blocks = [];
+  const summary = {};
+  return await Promise.map(repos, async ({ org, name }) => {
+    const alerts = await getVulnerabilities(org, name);
+    const criticalAlerts = _.filter(alerts, {
```

Comment:
I try to limit lodash in favor for es6. I think this could be rewritten as:
```
const criticalAlerts = alerts.filter((alert) => alert.severity === "critical" && !alert.dismissed)
```